packages/terra-form/docs/Input.md

```diff
@@ -0,0 +1,16 @@
+# Input
+
+Generic input which represents an HTML input element directly.
+
+## Usage
+
+```jsx
+import React from 'react';
+import { Input } from 'terra-form';
+
+<Input
+  required={false}
```

Comment:
I did see these standards where they wanted to mirror the DOM whenever it was possible. https://github.cerner.com/terra/conventions/blob/master/react.md#mirror-dom-wherever-possible

I'll look through war room agendas to see if we had updates.